src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Result/TestResult.cs

```diff
@@ -12,13 +12,20 @@ public enum TestResult
         /// The test completed successfully.
         /// </summary>
         Pass,
+
         /// <summary>
         /// The test failed and should be investigated.
         /// </summary>
         Fail,
+
+        /// <summary>
+        /// The test was skipped (<see cref="Validator.ShouldRunAsync(ValidationContext)"/> returned <see cref="ShouldRunTestResult.No"/>).
+        /// </summary>
+        Skip,
+
         /// <summary>
-        /// The test was skipped (<see cref="Validator.ShouldRunAsync(ValidationContext)"/> threw or returned false).
+        /// The result of the test could not be determined (<see cref="Validator.ShouldRunAsync(ValidationContext)"/> returned <see cref="ShouldRunTestResult.Unknown"/>)
```

Comment:
What do you think of a name like `Pending` to correspond to `ShouldRunTestResult.RetryLater`? If so, I would change the comment to something like, `The result of the test has not been determined`

Reply:
I would also mention that this will need a new leaf to move to a `Pass`/`Fail` final state